tests/test_vjp.py

```diff
@@ -98,8 +98,11 @@ def test_multiple_measurements(self, tol, dev):
         dy = np.array([1.0, 2.0, 3.0])
         tape1.trainable_params = {1, 2, 3}
 
+        ham = qml.Hamiltonian(dy, [qml.PauliX(0), qml.PauliY(1), qml.PauliY(1)])
+        print(ham)
+
         with qml.tape.QuantumTape() as tape2:
-            ham = qml.Hamiltonian(dy, [qml.PauliX(0), qml.PauliY(1), qml.PauliY(1)])
+            ham
```

Comment:
Can we remove `print` and move `qml.Ham` back in the `QuantumTape` scope?

Reply:
These sneak prints!!!!

Reply:
done!